packages/serverpod/lib/src/database/adapters/postgres/database_connection.dart

```diff
@@ -366,7 +369,22 @@ class DatabaseConnection {
     Transaction? transaction,
     bool ignoreRows = false,
     bool simpleQueryMode = false,
+    Object? parameters,
```

Comment:
Suggestion: Use the `QueryParameters?` type.

Is there any reason not to use the defined type here? Seems like it helps us with a couple of cases inside the method as well.

Reply:
This should also remove the need to assert if the parameter is not what we expect, since the interface does not allow any other type.

Reply:
Done